CHANGES.md

```diff
@@ -1,6 +1,362 @@
 Release Changes List
 ====================
 
+
+version 1.29.0
+==============
+
+released December 15, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* compile times have been reduced for most platforms via the use of 'jemalloc'
+* reduced overheads and improved scalability of creating distributed arrays
+* made significant additions and improvements to the 'IO' module
+* added a new compilation error framework and improved several error messages
+* improved choices made by procedure overload resolution to improve usability
+* added an experimental 'weak pointer' type with the goal of getting user input
+* added a prototype mode to capture the generated assembly for a given routine
+* made numerous renamings, deprecations, and improvements to standard modules
+* fixed a number of user-reported bugs
+
+Packaging / Configuration Changes
+---------------------------------
+* made our 'Dockerfile' use the current source rather than a specific release
+* LLVM 14 is now required when using the LLVM back-end on a Mac OS X  
+  (see https://chapel-lang.org/docs/1.29/usingchapel/prereqs.html)
+* doxygen 1.8.17 is now required to make the compiler developer documentation  
+  (see https://chapel-lang.org/docs/1.29/usingchapel/prereqs.html)
+
+Semantic Changes / Changes to the Chapel Language
+-------------------------------------------------
+* zippered for-loops led by unbounded ranges are now considered infinite  
+  (see https://chapel-lang.org/docs/1.29/language/spec/ranges.html#iterating-over-unbounded-ranges)
+* untyped formals are now preferred to those requiring implicit conversions  
+  (see https://chapel-lang.org/docs/1.29/language/spec/procedures.html#better-argument-mapping)
+* unary negation of `uint` now results in a compilation error for all widths  
+  (see https://chapel-lang.org/docs/1.29/language/spec/expressions.html#unary-minus-operators)
+
+Syntactic / Naming Changes
+--------------------------
+* renamed `makeIndexBuffer()` on domains to `createIndexBuffer()`  
+  (see https://chapel-lang.org/docs/1.29/language/spec/domains.html#ChapelDomain.createIndexBuffer)
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* deprecated support for operator-based subtype comparisons  
+  (e.g., `type1 <= type2`)
```

Comment:
Worth mentioning the method version?

Reply:
I think you're suggesting:

```
* deprecated operator-based subtype comparisons in favor of `isSubtype()`
```

is that right?  (don't ask me to come up with the equivalent of `<=` in the e.g., I can't keep the direction straight :D ).

Reply:
I was wondering about the method replacement for `<=`, but also didn't know which way it goes, so I didn't suggest anything either :). Fine leaving as-is given that